ci/jenkins/test-vm.sh

```diff
@@ -220,7 +276,7 @@ function run_e2e_vms {
     configure_vm_agent
     echo "====== Running Antrea e2e Tests for VM ======"
     mkdir -p `pwd`/antrea-test-logs
-    go test -v -timeout=100m antrea.io/antrea/test/e2e -run=TestVMAgent --logs-export-dir `pwd`/antrea-test-logs -provider=remote -windowsVMs=${WIN_HOSTNAME} -linuxVMs=${LIN_HOSTNAME}
+    go test -v -timeout=100m antrea.io/antrea/test/e2e -run=TestVMAgent --logs-export-dir `pwd`/antrea-test-logs -provider=remote -windowsVMs=${WIN_HOSTNAMES[@]} -linuxVMs=${LIN_HOSTNAMES[@]}
```

Comment:
Did you verify this can pass multiple names to windowsVMs and linuxVMs? I guess it doesn't work.

Reply:
Okay. If we should use it as a string instead of array that solved our issue.